.github/workflows/pr.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         node-version: [16.x]
-        prisma-client-version: ['3.5']
```

Comment:
This should be testing past versions of prisma. 3.12 is the latest. This should be an array of all the prisma versions that we officially support.

Reply:
We have a test issue with old versions
https://github.com/AhmedElywa/nexus-prisma/runs/6109633889?check_suite_focus=true#step:8:446

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/4535142/164521097-9ab3fbce-1991-4599-bc22-0e98d1ff7e93.png">